src/filters/ColorMatrixFilters.ts

```diff
@@ -7,6 +7,8 @@ export function createColorMatrixFilter(key: string, matrix: number[]) {
       return key;
     }
```

Comment:
should be removed, extending ColorMatrix is enough

Reply:
those had their own name before, will keep them.
I m not even sure is possible with the different defualts. Without their type class they would serialize to ColorMatrix and then to the next deserialize wouldn't work anymore

Reply:
I want this removed because I think in the future it is a potential bug
Once we remove type getter this thing will become an artifact

Reply:
> I m not even sure is possible with the different defualts

??